### DISCUSSION TITLE
store and restore custom windows position

### DISCUSSION THREAD

**Discussion by cyayon** [rxhanson/RectanglePro-Community#10] [Ideas]
Hello, 

i would be great if Rectangle pro could store and restore custom windows positions like this tools : https://cordlessdog.com/stay/

thanks

**Comment by rxhanson** [comment]
I have this on my backlog, but I haven’t quite decided how I want to implement it or when I will plan to roll it in.

**Comment by agfirewire** [comment]
I am not sure if this is an effect of me moving from an intel based mac to a M1 pro mac, but I have noticed on my multi-monitor setups have issues retaining proper window position when I unplug/replug my laptop into my dock.  With my previous intel based macs this would happen once and awhile, but now it is happening frequently enough that it is wrecking my workflow.

I am also coming from Optimal Layout which is no longer supported (which is what led me to checking out rectangles), so part of me wonders if Optimal Layout was doing something in the background I was unaware of to help preserve window placement, or if it is something else.

**Comment by samoweb3** [comment]
@rxhanson I now see a feature to save windows positions in Rectangle Pro, but I couldn't find a way to restore the saved positions.
Any shortcut available or any other way to restore the saved layout? ty
